### ISSUE TITLE
Compile bugs moving from 0.9.7 to 0.9.8

### ISSUE THREAD

**Issue by cjorasch** [ionic-team/stencil#864]
**Stencil version:**
```
 @stencil/core@0.9.8
```

**I'm submitting a:**
[x] bug report

**Current behavior:**
When I upgraded from 0.9.7 to 0.9.8 my application stopped running due to `const enum` values.

For example:

```tsx
// my-enum.ts
export const enum MyEnum {
    A = 1,
    B = 2,
    C = 3
}

// my-component.ts
import { MyEnum } from './my-enum';

if (value === MyEnum.A) { ... }
```

When building with 0.9.7 the following code is output:

```tsx
// my-component.js
if (value === 1 /* A */) { ... }
```

When building with 0.9.8 the following code is output:

```tsx
// my-component.js
if (value === MyEnum.A) { ... }
```

There is no import and the value has not been replaced so the code fails since `MyEnum` is not defined.

**Comment by jthoms1** [comment]
I would assume this is a change from TypeScript 2.8 to 2.9.  I didn't find any bug reports but if you set `preserveConstEnums: true` in your tsconfig file it should emit the enums.

https://www.typescriptlang.org/docs/handbook/compiler-options.html

**Comment by adamdbradley** [comment]
Should be fixed now, thanks! https://github.com/ionic-team/stencil/commit/2b71d1553da6a9335c0d30ffa734f33d547dadef

**Comment by cjorasch** [comment]
Confirmed fix in 0.9.9.  Thanks!

Ran into another problem with chunks that have classes in wrong order and failing to run.  Will try to diagnose.
